.github/workflows/testrunner.yml

```
@@ -12,7 +12,7 @@ jobs:
   build-and-publish:
     uses: ./.github/workflows/build-and-publish-image.yml
     with:
-      tags: 'ghcr.io/datadog/dd-trace-py/testrunner:${{ github.sha }},ghcr.io/datadog/dd-trace-py/testrunner:latest'
+      tags: 'ghcr.io/datadog/dd-trace-py/testrunner:${{ github.sha }},ghcr.io/datadog/dd-trace-py/testrunner:0f49c377e3568020d82e8fa2b4f314900deb8f8d'
```

don't use latest here, this might clash with 2.x releases

```suggestion
      tags: 'ghcr.io/datadog/dd-trace-py/testrunner:${{ github.sha }}'
```